README.md

```diff
@@ -147,6 +147,45 @@ resources:
         newKey: user_id
 ```
 
+### Batch Resources with `properties` paramerters
+
+Instead of accepting just a list of users (`user_ids`), a batch resouce could accept both a list of users (`user_ids`) and a list of properties (`properties`):
+
+```js
+const getUserInfo = (args: { user_ids: Array<number>, properties: Array<string> }): Promise<Array<UserInfo>> =>
+    fetch('/userInfo', args);
```

Comment:
```suggestion
const users = getUsers({
  ids: [1, 2, 3],
  properties: ['firstName', 'age'],
})

/**
 * e.g. users => 
 * [
 *   { firstName: 'Alice', age: 42 },
 *   { firstName: 'Bob', age: 70 }
 * ]
 */
```

Just to clarify with example query and importantly, response

Reply:
I keep the line 155 and 156, I think the definition of the resource is also the part of this example.

Reply:
sounds good